Sources/armory/logicnode/GetCursorStateNode.hx

```diff
@@ -12,13 +12,13 @@ class GetCursorStateNode extends LogicNode {
 		var mouse = iron.system.Input.getMouse();
 
 		return switch (from) {
-		case 0:
-		if (mouse.hidden == true && mouse.locked == true) return true;
-		return false;
+			case 0:
+			if (mouse.hidden == true && mouse.locked == true) return true;
+			return false;
 
-		case 1: mouse.hidden;
-		case 2: mouse.locked;
-    default: null;
-    }
+			case 1: mouse.hidden;
+			case 2: mouse.locked;
+    			default: null;
```

Comment:
Again some problem with tabs and spaces (sorry for being so nitpicky here)

Reply:
No problem about nitpicky, i also prefer the things the better as possible